lib/sycamore/sycamore/tests/unit/transforms/compare_detr_impls.py

```diff
@@ -11,6 +11,18 @@ def compare_batched_sequenced(partitioner, path, **kwargs):
     return batched
 
 
+def check_table_extraction(partitioner, path, **kwargs):
+    with open(path, "rb") as f:
+        sequenced = partitioner._partition_pdf_sequenced(f, **kwargs)
+        hash_key = Cache.get_hash_context(f.read()).hexdigest()
+    batched = partitioner._partition_pdf_batched_named(path, hash_key, **kwargs)
```

Comment:
So long as we are running both, might as well add 
`assert deep_eq(batched, sequenced)`

Reply:
Done